controllers/mongodb_tls.go

```diff
@@ -90,7 +87,7 @@ func getTLSConfigModification(getUpdateCreator secret.GetUpdateCreator, mdb mdbv
 		return automationconfig.NOOP(), nil
 	}
 
-	certKey, err := getCertAndKey(getUpdateCreator, mdb)
+	certKey, err := getPemOrConcatCrtAndKey(getUpdateCreator, mdb)
```

Comment:
I think the name `getPemOrConcatCrtAndKey` is a bit confusing , maybe we can remove the `OrConcatCrtAndKey` since it is an implementation details of the function ?

Reply:
I called it `getFinalPem` earlier but I didn't like it. We already have `getCertAndKey` and `getPem` so I wanted a name that was clearly different.

Reply:
I see in that case maybe s/Concat/Concatenated? since the current signature, it looks like the function is performing concatenation -- will leave it upto you